### PR TITLE
Adapt response handling in async runners

### PR DESCRIPTION
**Pull request by danielmitterdorfer**
With this commit we adapt the response handling in async runners by
properly converting them from raw bytes to strings.

Relates elastic/rally#944

### CODE REVIEW

File: eventdata/runners/deleteindex_runner.py [hunk 1]
```diff
@@ -66,7 +66,8 @@ def get_suffix(name, separator):
     suffix_separator = params.get('suffix_separator', '-')
 
     if max_indices:
-        indices = await es.cat.indices(h='index').split("\n")
+        response = await es.cat.indices(h='index')
```
Comment (hub-cap) [inline eventdata/runners/deleteindex_runner.py:69]: should this be double quotes here as well?

Comment (danielmitterdorfer) [reply eventdata/runners/deleteindex_runner.py:69]: Good point. I tried to change as little as possible and the eventdata track is quite inconsistent in any case but let's change this here.

Comment (danielmitterdorfer) [reply eventdata/runners/deleteindex_runner.py]: I've pushed https://github.com/elastic/rally-eventdata-track/pull/82/commits/07332a787a9c056205cae5f5ebe2cb6a8ae89aac.

**Comment by danielmitterdorfer** [comment]
Thanks for the review!

**Pull request merged**